mobile/biome.json

```
@@ -0,0 +1,7 @@
+{
+  "$schema": "https://biomejs.dev/schemas/2.0.0-beta.3/schema.json",
+  "extends": ["@still-forest/biomejs-config"],
+  "files": {
+    "includes": ["!ios/**", "!android/**", "!node_modules/**", "!dist/**"]
```

Biome (1.9.4)

[error] Found an unknown key `includes`.

Known keys:


(deserialize)